config.json

```diff
@@ -69,12 +69,14 @@
    ,"saddle-points"
    ,"poker"
    ,"dominoes"
+   ,"all-your-base"
   ],
   "deprecated": [
 
   ],
   "ignored": [
     "docs",
+    "img",
```

Comment:
This image was already merged? Maybe that is why Travis is failing. Also the doct update was already merged